internal/runner/runner.go

```diff
@@ -249,7 +249,29 @@ func getTool(toolName string, tools []types.Tool) *types.Tool {
 }
 
 func requirementSatisfied(requirementName string) bool {
+	if strings.HasPrefix(requirementName, "lib") {
+		libNames := appendLibExtensionForOS(requirementName)
+		for _, libName := range libNames {
+			_, sysErr := syscallutil.LoadLibrary(libName)
+			if sysErr == nil {
+				return true
+			}
+		}
+		return false
+	}
 	_, execErr := exec.LookPath(requirementName)
-	_, sysErr := syscallutil.LoadLibrary(requirementName)
-	return sysErr == nil || execErr == nil
+	return execErr == nil
+}
+
+func appendLibExtensionForOS(lib string) []string {
```

Comment:
already more than readable - but in case you might find it useful https://github.com/projectdiscovery/utils/blob/main/os/os.go